CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ link_libraries(
 if (GSTREAMER_FOUND)
   link_libraries(
     ${GSTREAMER_LIBRARIES}
+    gstapp-1.0
```

Comment:
Probably I just don't understand gstreamer well enough, but do you why this is not part of     `${GSTREAMER_LIBRARIES}`?

Reply:
@Jaeyoung-Lim I don't follow QGC close last time. As far as I see they try to use newer gsteramer and there could be some incompatibilities. We still use 1.14 on our side and everything works. I feel like I'll need to upgrade gstreamer on our side so perhaps I'll also tweak QGC at this time and it will be as stable as it was.

Regarding gstapp-1.0 - it is part of gstreamer-app-1.0 pkg-config's package. Looking on cmake/FindGStreamer.cmake I see that we can try something like FIND_GSTREAMER_COMPONENT(GSTREAMER_APP gstreamer-app-1.0 gstapp-1.0) which probably could be used to pickup that library in more fancy way and then it will become part of ${GSTREAMER_APP_LIBRARIES}. I don't see big sense in that since this library is part of base plugins which should be always installed (at least they were used until now without explicitly detecting if they are available - I did exactly the same).

Reply:
Thanks for the explanations!

Reply:
Actually I was wrong - the way how brew (CI) installs gstreamer packages on macOS results in different header layout than on typical *nix system (or after official gstreamer package). We have to use cmake/FindGStreamer.cmake to pickup everything. I'm trying to solve that https://github.com/PX4/PX4-SITL_gazebo/pull/759